Bottom: 1d5006aadc74ad1ef44c5f5a451e8a0c1957dafb
Top:    302d33d5bb32c5145371cfd44c4a21d5852e6c1f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 19:46:15 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/find.c b/src/find.c
index ffbde1f..274a0f5 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3322,16 +3322,15 @@ LookupUnusedPins (FILE * FP)
 bool
 ResetFoundPinsViasAndPads (bool AndDraw, int flag)
 {
-#warning OBEY THE FLAG
   bool change = false;
 
   VIA_LOOP (PCB->Data);
   {
-    if (TEST_FLAG (TheFlag, via))
+    if (TEST_FLAG (flag, via))
       {
         if (AndDraw)
           AddObjectToFlagUndoList (VIA_TYPE, via, via, via);
-        CLEAR_FLAG (TheFlag, via);
+        CLEAR_FLAG (flag, via);
         if (AndDraw)
           DrawVia (via);
         change = true;
@@ -3342,11 +3341,11 @@ ResetFoundPinsViasAndPads (bool AndDraw, int flag)
   {
     PIN_LOOP (element);
     {
-      if (TEST_FLAG (TheFlag, pin))
+      if (TEST_FLAG (flag, pin))
         {
           if (AndDraw)
             AddObjectToFlagUndoList (PIN_TYPE, element, pin, pin);
-          CLEAR_FLAG (TheFlag, pin);
+          CLEAR_FLAG (flag, pin);
           if (AndDraw)
             DrawPin (pin);
           change = true;
@@ -3355,11 +3354,11 @@ ResetFoundPinsViasAndPads (bool AndDraw, int flag)
     END_LOOP;
     PAD_LOOP (element);
     {
-      if (TEST_FLAG (TheFlag, pad))
+      if (TEST_FLAG (flag, pad))
         {
           if (AndDraw)
             AddObjectToFlagUndoList (PAD_TYPE, element, pad, pad);
-          CLEAR_FLAG (TheFlag, pad);
+          CLEAR_FLAG (flag, pad);
           if (AndDraw)
             DrawPad (pad);
           change = true;
@@ -3379,16 +3378,15 @@ ResetFoundPinsViasAndPads (bool AndDraw, int flag)
 bool
 ResetFoundLinesAndPolygons (bool AndDraw, int flag)
 {
-#warning OBEY THE FLAG
   bool change = false;
 
   RAT_LOOP (PCB->Data);
   {
-    if (TEST_FLAG (TheFlag, line))
+    if (TEST_FLAG (flag, line))
       {
         if (AndDraw)
           AddObjectToFlagUndoList (RATLINE_TYPE, line, line, line);
-        CLEAR_FLAG (TheFlag, line);
+        CLEAR_FLAG (flag, line);
         if (AndDraw)
           DrawRat (line);
         change = true;
@@ -3397,11 +3395,11 @@ ResetFoundLinesAndPolygons (bool AndDraw, int flag)
   END_LOOP;
   COPPERLINE_LOOP (PCB->Data);
   {
-    if (TEST_FLAG (TheFlag, line))
+    if (TEST_FLAG (flag, line))
       {
         if (AndDraw)
           AddObjectToFlagUndoList (LINE_TYPE, layer, line, line);
-        CLEAR_FLAG (TheFlag, line);
+        CLEAR_FLAG (flag, line);
         if (AndDraw)
           DrawLine (layer, line);
         change = true;
@@ -3410,11 +3408,11 @@ ResetFoundLinesAndPolygons (bool AndDraw, int flag)
   ENDALL_LOOP;
   COPPERARC_LOOP (PCB->Data);
   {
-    if (TEST_FLAG (TheFlag, arc))
+    if (TEST_FLAG (flag, arc))
       {
         if (AndDraw)
           AddObjectToFlagUndoList (ARC_TYPE, layer, arc, arc);
-        CLEAR_FLAG (TheFlag, arc);
+        CLEAR_FLAG (flag, arc);
         if (AndDraw)
           DrawArc (layer, arc);
         change = true;
@@ -3423,11 +3421,11 @@ ResetFoundLinesAndPolygons (bool AndDraw, int flag)
   ENDALL_LOOP;
   COPPERPOLYGON_LOOP (PCB->Data);
   {
-    if (TEST_FLAG (TheFlag, polygon))
+    if (TEST_FLAG (flag, polygon))
       {
         if (AndDraw)
           AddObjectToFlagUndoList (POLYGON_TYPE, layer, polygon, polygon);
-        CLEAR_FLAG (TheFlag, polygon);
+        CLEAR_FLAG (flag, polygon);
         if (AndDraw)
           DrawPolygon (layer, polygon);
         change = true;
